js/whitebit.js

```diff
@@ -462,23 +463,26 @@ module.exports = class whitebit extends Exchange {
         //           {...}
         //      }
         //
-        const currenciesIds = Object.keys (response);
-        const withdrawFees = {};
-        const depositFees = {};
-        for (let i = 0; i < currenciesIds.length; i++) {
-            const currency = currenciesIds[i];
+        const result = {};
+        if (codes === undefined) {
+            codes = Object.keys (response);
+        }
+        for (let i = 0; i < codes.length; i++) {
+            const currency = codes[i];
             const data = response[currency];
             const code = this.safeCurrencyCode (currency);
+            if (codes !== undefined && !this.inArray (code, codes)) {
```

Comment:
@AlgoQ I don't think we need this if anymore, first `codes` will never be undefined here, you're iterating over it, second if the user does not provide any currency you're getting the currencies from the response so they will always be there, if he does and passes the `safeCurrencyCode` check the currency will always exist as well